README.md

```diff
@@ -24,10 +24,13 @@
 - LinkedIn: [@silviatofana](www.linkedin.com/in/silvia-tofana-10b852186)
 - Twitter: [@silviatofana](https://twitter.com/SilviaTofana)
 
--
-## Collaborators
+👤 Grace Muthui
 
+- GitHub: [@Grayce Muthui](https://github.com/Graycemuthui)
+- Twitter: [@Grayce_Muthui](https://twitter.com/Grayce_Muthui)
+- LinkedIn: [@Grayce Muthui](http://www.linkedin.com/in/grayce-muthui-a17294226)
 
+## Collaborators
 
 ## 🤝 Contributing
```

Comment:
- [ ] I would recommend including the steps on how to test the application in the README file. [Hint: npm test]

Reply:
ok, thanks for the helpful advice I will include in the README file how to test the application. [Hint: npm test]